### ISSUE TITLE
Collection/libraries tree: poor experience with macOS voice over

### ISSUE THREAD

**Issue by tnajdek** [zotero/web-library#498]
VoiceOver  ignores the fact that active element is of `role="treeitem"` and has accessible name set correctly, it instead reports it's a "cell" in a "table" and does not announce the label.

I suspect this might have behaved better before macOS 13.x.

This might be an issue with VoiceOver and not our markup: https://twitter.com/tcaspers/status/1420466213315096582
